numerics/apodization.hpp

```diff
@@ -15,9 +18,56 @@ template<template<typename, typename, int> class Evaluator>
 PoissonSeries<double, 0, Evaluator> Dirichlet(Instant const& t_min,
                                               Instant const& t_max);
 
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> Sine(Instant const& t_min,
+                                         Instant const& t_max);
+
+template<template<typename, typename, int> class Evaluator>
+PoissonSeries<double, 0, Evaluator> Hahn(Instant const& t_min,
```

Comment:
[Hann](https://en.wikipedia.org/wiki/Julius_von_Hann) is not [Hahn](https://en.wikipedia.org/wiki/Hans_Hahn_(mathematician)).